packages/renderer/src/lib/preferences/PreferencesProxiesRendering.svelte

```diff
@@ -87,12 +95,13 @@ function validate(event: any) {
       <div class="space-y-2">
         <label
           for="httpsProxy"
-          class="pt-4 mb-2 font-medium {proxyState ? 'text-[var(--pd-invert-content-card-text)]' : 'text-gray-900'}"
-          >Secure Web Proxy (HTTPS):</label>
+          class="pt-4 mb-2 font-medium {proxyState === 1
+            ? 'text-[var(--pd-invert-content-card-text)]'
+            : 'text-gray-900'}">Secure Web Proxy (HTTPS):</label>
         <Input
           name="httpsProxy"
           id="httpsProxy"
-          disabled={!proxyState}
+          disabled={proxyState !== 1}
```

Comment:
```suggestion
          disabled={proxyState !== ProxyState.PROXY_MANUAL}
```

Copy paste from line 86